app/src/styles/base.css

```
@@ -200,6 +200,234 @@ h1, h2, h3, h4 { font-family: var(--font-heading); margin: 0; }
+}
+.live-dot {
+  width: 6px;
+  height: 6px;
+  flex: none;
+  border-radius: 999px;
+  background: var(--up);
+  /* Breathing, not blinking: a hard on/off in the corner of a screen full of
+   * numbers reads as an error light. Opacity only. */
+  animation: livePulse 1.8s ease-in-out infinite;
+}
+@keyframes livePulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "livePulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)